lib/active_model/serializer/adapter/cached_serializer.rb

```diff
@@ -24,7 +24,7 @@ def cached?
         end
 
         def fragment_cached?
-          @klass._cache_only && !@klass._cache_except || !@klass._cache_only && @klass._cache_except
+          @klass._cache && (@klass._cache_only && !@klass._cache_except || !@klass._cache_only && @klass._cache_except)
```

Comment:
I guess the idea was to check if fragment_caching was available independent of whether or not caching is available